set.mm

```diff
@@ -735799,6 +735836,340 @@ their group (addition) operations are equal for all pairs of elements of
+    gsumsplit2f.i $e |- ( ph -> ( C i^i D ) = (/) ) $.
+    gsumsplit2f.u $e |- ( ph -> A = ( C u. D ) ) $.
+    $( Split a group sum into two parts.  (Contributed by AV, 4-Sep-2019.) $)
+    gsumsplit2f $p |- ( ph -> ( G gsum ( k e. A |-> X ) ) =
+      ( ( G gsum ( k e. C |-> X ) ) .+ ( G gsum ( k e. D |-> X ) ) ) ) $=
+      ( cmpt cgsu cres eqid fmptdf gsumsplit cun ssun1 sseqtrrid resmptd oveq2d
+      co ssun2 oveq12d eqtrd ) AHGBJUBZUCUMHUQDUDZUCUMZHUQEUDZUCUMZFUMHGDJUBZUC
+      UMZHGEJUBZUCUMZFUMABCDEFUQHIKMNOPQAGBJCUQLRUQUEUFSTUAUGAUSVCVAVEFAURVBHUC
+      AGBDJADEUHZDBDEUIUAUJUKULAUTVDHUCAGBEJAVFEBEDUNUAUJUKULUOUP $.
+  $}
+
+  ${
+    $d k A $.  $d k B $.  $d k G $.  $d k M $.
+    gsumdifsndf.k $e |- F/_ k Y $.
+    gsumdifsndf.n $e |- F/ k ph $.
+    gsumdifsndf.b $e |- B = ( Base ` G ) $.
+    gsumdifsndf.p $e |- .+ = ( +g ` G ) $.
+    gsumdifsndf.g $e |- ( ph -> G e. CMnd ) $.
+    gsumdifsndf.a $e |- ( ph -> A e. W ) $.
+    gsumdifsndf.f $e |- ( ph -> ( k e. A |-> X ) finSupp ( 0g ` G ) ) $.
+    gsumdifsndf.e $e |- ( ( ph /\ k e. A ) -> X e. B ) $.
+    gsumdifsndf.m $e |- ( ph -> M e. A ) $.
+    gsumdifsndf.y $e |- ( ph -> Y e. B ) $.
+    gsumdifsndf.s $e |- ( ( ph /\ k = M ) -> X = Y ) $.
+    $( Extract a summand from a finitely supported group sum.  (Contributed by
+       AV, 4-Sep-2019.) $)
+    gsumdifsndf $p |- ( ph -> ( G gsum ( k e. A |-> X ) )
+                        = ( ( G gsum ( k e. ( A \ { M } ) |-> X ) ) .+ Y ) ) $=
+      ( cmpt cgsu co csn cdif c0g cfv eqid c0 cin difid wss wceq snssd syl5reqr
+      difin2 syl cun wcel difsnid eqcomd gsumsplit2f ccmn cmnmnd gsumsnfd eqtrd
+      cmnd oveq2d ) AFEBIUBUCUDFEBGUEZUFZIUBUCUDZFEVJIUBUCUDZDUDVLJDUDABCVKVJDE
+      FHIFUGUHZLMVNUINOPRQAUJVJVJUFZVKVJUKZVJULAVJBUMVOVPUNAGBSUOVJVJBUQURUPAVK
+      VJUSZBAGBUTVQBUNSBGVAURVBVCAVMJVLDAICJEFGBMAFVDUTFVHUTOFVEURSTUALKVFVIVG
+      $.
+  $}
+
+  ${
+    gsumreidx.b $e |- B = ( Base ` G ) $.
+    gsumreidx.z $e |- .0. = ( 0g ` G ) $.
+    gsumreidx.g $e |- ( ph -> G e. CMnd ) $.
+    gsumreidx.f $e |- ( ph -> F : ( M ... N ) --> B ) $.
+    gsumreidx.h $e |- ( ph -> H : ( M ... N ) -1-1-onto-> ( M ... N ) ) $.
+    $( Re-index a finite group sum using a bijection.  Corresponds to the first
+       equation in [Lang] p. 5 with ` M = 1 ` .  (Contributed by AV,
+       26-Dec-2023.) $)
+    gsumreidx $p |- ( ph -> ( G gsum F ) = ( G gsum ( F o. H ) ) ) $=
+      ( cfz co cvv ovexd fzfid wcel c0g fvexi a1i fdmfifsupp gsumf1o ) AFGNOZBU
+      ECDEPHIJKAFGNQLAUEBCPHLAFGRHPSAHDTJUAUBUCMUD $.
+  $}
+
+  ${
+    gsumfsupp.b $e |- B = ( Base ` G ) $.
+    gsumfsupp.z $e |- .0. = ( 0g ` G ) $.
+    gsumfsupp.s $e |- I = ( F supp .0. ) $.
+    gsumfsupp.g $e |- ( ph -> G e. CMnd ) $.
+    gsumfsupp.a $e |- ( ph -> A e. V ) $.
+    gsumfsupp.f $e |- ( ph -> F : A --> B ) $.
+    gsumfsupp.w $e |- ( ph -> F finSupp .0. ) $.
+    $( A group sum is identical with the group sum restricted to the finite
```

Comment:
```suggestion
    $( A group sum is identical with the group sum restricted to the
```

Reply:
Why should "finite" be removed? It corresponds to the hypothesis `( ph -> F finSupp .0. )`, and I think that this hypothesis is necessary.

Reply:
I should have explained indeed. We restrict the sum to "the support of ..." which is a notion defined earlier. That support happens to be finite (and it is a necessary hypothesis indeed). "Having finite support" is a property which is also defined earlier.  However, there is no such thing as "the finite support of" a family.

Also, I missed that you wrote "support of the index set" but, it's actually the support of the family.

So, the comment could be phrased:
> The group sum of a family can be restricted to the support of that family without changing its value, provided that that support is finite."

(no typo, "that" appears twice).

Could you prove the same statement in a unital magma ?

Reply:
> Could you prove the same statement in a unital magma ?
I thought about this, too. But I think a lot of preparatory work is necessary for this.